src/common/sourceMaps/sourceMapFactory.test.ts

```diff
@@ -43,11 +44,26 @@ const unresolvedIndexedSourceMap: RawIndexMapUnresolved = {
 
 describe('SourceMapFactory', () => {
   let stubDap: StubDapApi;
-  let factory: SourceMapFactory;
+  let factory: RootSourceMapFactory;
+  let resourceProvider: IResourceProvider;
 
   beforeEach(() => {
     stubDap = stubbedDapApi();
-    factory = new SourceMapFactory(
+    resourceProvider = {
+      fetch(url) {
+        return Promise.resolve({
+          ok: true,
+          body: dataUriToBuffer(url).toString('utf8'),
+          url: url,
+          statusCode: 500,
```

Comment:
Is 500 intentional in this case?

Reply:
not really, but just test code, doesn't matter too much 😛